Core/PSPLoaders.cpp

```diff
@@ -83,7 +83,7 @@ void InitMemoryForGameISO(FileLoader *fileLoader) {
 
 	IFileSystem *fileSystem = nullptr;
 	IFileSystem *blockSystem = nullptr;
-	bool actualIso = false;
+	//bool actualIso = false;
```

Comment:
Let's just remove the line.  Git remembers, no need to keep commented out code.

-[Unknown]